bin/run.py

```diff
@@ -127,29 +129,33 @@ def __init__(self, problem, path, skip_double_build_warning=False):
         # interactor exits first. Thus, we don't distinguish the different non-AC
         # verdicts.
         if self.problem.interactive and (is_windows() or is_bsd()):
-            wrong_verdicts = ['WRONG_ANSWER', 'TIME_LIMIT_EXCEEDED', 'RUN_TIME_ERROR']
+            wrong_verdicts = [
+                Verdict.WRONG_ANSWER,
+                Verdict.TIME_LIMIT_EXCEEDED,
+                Verdict.RUNTIME_ERROR,
+            ]
             for wrong_verdict in wrong_verdicts:
                 if wrong_verdict in self.expected_verdicts:
                     self.expected_verdicts += wrong_verdicts
                     break
 
-    def _get_expected_verdicts(self):
-        verdicts = []
+    def _get_expected_verdicts(self) -> list[Verdict]:
+        expected_verdicts = []
 
         # Look for '@EXPECTED_RESULTS@: ' in all source files. This should be followed by a comma separated list of the following:
         # - ACCEPTED / CORRECT
         # - WRONG_ANSWER / WRONG-ANSWER / NO-OUTPUT
         # - TIME_LIMIT_EXCEEDED / TIMELIMIT
         # - RUN_TIME_ERROR / RUN-ERROR
-        domjudge_verdict_map = {
-            'CORRECT': 'ACCEPTED',
-            'WRONG-ANSWER': 'WRONG_ANSWER',
-            'TIMELIMIT': 'TIME_LIMIT_EXCEEDED',
-            'RUN-ERROR': 'RUN_TIME_ERROR',
-            'NO-OUTPUT': 'WRONG_ANSWER',
-            'CHECK-MANUALLY': None,
-            'COMPILER-ERROR': None,
-        }
+        domjudge_verdicts = [
+            'CORRECT',
+            'WRONG-ANSWER',
+            'TIMELIMIT',
+            'RUN-ERROR',
+            'NO-OUTPUT',
+            #'CHECK-MANUALLY', TODO this is *not* supported, shouldn't pretend it is
+            'COMPILER-ERROR',
+        ]
```

Comment:
This list is not needed anymore, now that we have `verdicts.from_string`, right?

Reply:
Removed in 8e91a088c. Fingers crossed.